CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
   If needed, `UnidirectionalTopicOperator` can be disabled in the feature gates configuration in the Cluster Operator.
 * Improved Kafka Connect metrics and dashboard example files
 * Allow specifying and managing KRaft metadata version
+* Add support for KRaft to KRaft upgrades (Apache Kafka upgrades for the KRaft based clusters)
```

Comment:
Should we add the min supported Strimzi version in this note?

Reply:
What do you mean with `min supported Strimzi version`?

Reply:
We handle the case where pods are there but without kafka-version annotation, which results in KafkaUpgradeException. So I think adding a note to avoid that experience would be good (for users that actually read release notes and documentation).

Reply:
Sorry, I'm still not sure what is it you are asking for.